cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,68 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Settings page', () => {
-  before(() => {
-
-  });
+import settingsPageObject from '../support/pages/settings.pageObject.js';
+const settingsPage = new settingsPageObject();
 
-  beforeEach(() => {
+describe('Settings page', () => {
+    let user;
+    beforeEach(() => {
+        cy.task('db:clear');
+        cy.task('generateUser').then((generateUser) => {
+            user = generateUser;
+            cy.login(user.email, user.username, user.password);
+        });
+        cy.visit(settingsPage.url);
+    });
 
-  });
 
   it('should provide an ability to update username', () => {
+    const newUsername = 'AdolfWhiteCat';
+    settingsPage.usernameField.clear()
+    .type(newUsername);
+    settingsPage.updateBtn.click();
 
+    settingsPage.usernameField.should('have.value', newUsername);
```

Comment:
```suggestion
    settingsPage.usernameField.should('have.value', newUsername);
```
wrap this and similar methods inside page object.